docs/initial-setup.rst

```diff
@@ -160,9 +167,14 @@ command::
 
     fab <environment> update_server_passwords
 
-**Note:** It's only necessary to have a copy of ``fabsecrets_<environment>.py`` locally if you
-will be deploying new servers or updating the existing passwords on the
-servers.
+.. NOTE::
+   It's only necessary to have a copy of ``fabsecrets_<environment>.py`` locally if you
+   will be deploying new servers or updating the existing passwords on the servers.
+
+.. NOTE::
+   This command is really only useful on the web and worker servers. On all other servers,
+   the command will update the secrets, but nothing will force the running processes to re-read
```

Comment:
I think it'd be more correct to say that nothing will update the configuration files on the systems to use the new secrets.